she_codes_news/news/models.py

```diff
@@ -1,8 +1,25 @@
+from django.contrib.auth import get_user_model
 from django.db import models
 
 
-class NewsStory(models.Model):
+class NewsStory(models.Model):     
+    class Meta:
+        ordering = ["-pub_date"]
+        verbose_name_plural = "News Stories"
+        
     title = models.CharField(max_length=200)
-    author = models.CharField(max_length=200)
+    author = models.ForeignKey(get_user_model(), on_delete=models.CASCADE, related_name="stories")
     pub_date = models.DateTimeField()
     content = models.TextField()
+    image = models.URLField(max_length=200, blank=True) 
+    favourited_by = models.ManyToManyField(
+        get_user_model(), related_name= "favourites", blank=True
+        )
+    
```

Comment:
would have been very impressed if i had seen a category = foerignkey field here linking to the category model below!